odoo/tests/common.py

```diff
@@ -1235,7 +1235,7 @@ def _process_fvg(self, model, fvg, level=2):
             order.append(fname)
 
             modifiers[fname] = {
-                modifier: domain if isinstance(domain, bool) else normalize_domain(domain)
+                modifier: domain if isinstance(domain, int) else normalize_domain(domain)
```

Comment:
Alternatively,
```suggestion
                modifier: bool(domain) if isinstance(domain, int) else normalize_domain(domain)
```
would mean we don't have to touch the second line

Reply:
You're right! Thanks a lot, done.

CC @lescobarvx